hooks/get_published_files.py

```diff
@@ -199,5 +172,5 @@ def get_latest_published_file(self, item, data_retriever=None, **kwargs):
                 fields=fields,
                 order=order,
             )
-
+        self.logger.debug("Found latest Published File %s" % result)
```

Comment:
Same comment: in one case it's SG records, in the other it's a bg task id, I think?